Doc/library/pathlib.rst

```diff
@@ -7,7 +7,7 @@
 
 .. versionadded:: 3.4
 
-**Source code:** :source:`Lib/pathlib.py`
+**Source code:** :source:`Lib/pathlib/`
```

Comment:
```suggestion
**Source code:** :source:`Lib/pathlib.py`
```
On 3.12 it was still one file: https://github.com/python/cpython/blob/3.12/Lib/pathlib.py